### ISSUE TITLE
MAUI 7.0 iOS Release build won't install on iOS 16

### ISSUE THREAD

**Issue by jdunaway** [dotnet/maui#11838]
### Description

Please see original issue which was closed: https://github.com/dotnet/maui/issues/11385  Re-opening with a reproduction repo since the other issue was closed and I don't have permission to reopen it.  I was able to reproduce the same issue with a File -> New MAUI project (please see "Link to public reproduction project repository").   Please see the azure-pipelines.yml file in the reproduction repo for how I'm building this. I've also tried building this locally on the windows and mac side but I'm currently getting a Link.semaphore error. So you may want to try building this locally too. You will need to update the certificate to an adhoc iOS distribution certificate and update the provisioning profile to your own provisioning profile in order to reproduce.

I'm using Azure DevOps to create an ad-hoc build for iOS for a .net 7 MAUI application (recently updated from .net 6 which had the same issue):

`dotnet publish -f net7.0-ios -c Release -r ios-arm64 /p:ArchiveOnBuild=true `

The .ipa file produced by the build works on iOS 15 devices but when I download it on an iOS 16 device the icon is blank and when I tap it to open it says the application cannot be installed at this time. Has anyone successfully published a MAUI release build for iOS 16?

### Steps to Reproduce

1. publish ad-hoc release build of .net maui application `dotnet publish -f net7.0-ios -c Release -r ios-arm64 /p:ArchiveOnBuild=true `
2. Download it on an iOS 16 device.  
3. For me, the icon is blank and when I click the application I see a message saying "Unable to Install  Please try again later."

### Link to public reproduction project repository

https://github.com/jdunaway/MauiAppReproiOS16ReleaseBuildIssue

### Version with bug

7.0 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

iOS

### Affected platform versions

iOS 16

### Did you find any workaround?

No

**Comment by epsmae** [comment]
@jdunaway yes we have upgraded our productive apps to net 7 and they work on ios 16.1.1.
We had some styling issues and needed to adjust some stuff but no crashes.

**Comment by jdunaway** [comment]
Closing this since the original issue https://github.com/dotnet/maui/issues/11385 has been reopened.
